\documentclass[../main.tex]{subfiles}
\begin{document}

\newpage
\section{Introduction}

The mitochondrial genomes of non-bilaterian animals, such as sponges, ctenophores, and cnidarians hold the majority of the genetic diversity seen in Metazoan mitochondrial genomes. Of these, phylum Porifera (sponges) offers a unique perspective on ancient Metazoans and the subsequent evolution of key animal traits such as symmetry and tissue development. Their ancient evolutionary history, combined with their wide geographically range and availability make them useful for studying the genetic diversity in non-bilaterians. 

The loss of mitochondrial genes is seldom seen in bilaterian animals such as mammals, and is a much more common, though still rare, occurrence in non-bilaterians. More often, non-bilaterian genomes have additional components. One example is the repetitive invasion of hairpin loops seen in many sponge mitochondrial genomes. This invasion has happened independently in many lineages of sponges, and almost always appear to be recent in the organism's evolutionary history. The origin of this invasion is unknown and the respective function of the stem-loops has not been studied, though some hypothesize the stem-loops might function as regulatory mechanisms. The result is almost always a substantial increase in the size of mitochondrial genomes. In addition, \emph{atp9}, a gene that encodes for a subunit of the mitochondrial F0-ATP synthase, is found only in sponges, and has not been found in any other lineage outside of plants and the phylum Porifera. 

However, the most interesting and novel mt-genome abnormality seen in sponges is the sporadic loss of mt-tRNA genes across numerous lineages. This loss is of scientific interest, as it was originally believed that mt-tRNA genes were critical for normal cellular function. In particular, numerous human diseases are caused by mutations in human mt-tRNA genes. The discovery that some animals are able to lose some, or all, of their mitochondrial tRNA genes make them a prime target for study.

For all the sponge species with sequenced mitochondrial genomes, no pattern of mt-tRNA gene loss has been identified, and the impact the loss has on cellular processes and genome evolution has not been widely studied. Previous research has shown mitochondrial tRNA synthetases can be lost in parallel with mitochondrial tRNAs genes. However, this is not always the case. It does appear that certain mt-tRNA genes and mitochondrial tRNA synthetases are retained. The factors that promote this loss of tRNA genes and the mechanisms by which it occurs are also unknown. 

One of the most studied sponges, \emph{Amphimedon queenslandica}, showcases many of the above traits seen in sponge mt-genomes. It is considerably longer than the standard Metazoan mt-genome, has long non-coding regions, displays many stem-loop elements both within and outside protein-coding genes, and has no identifiable control region. Most notable in the \emph{Amphimedon queenslandica} mt-genome is the loss of the gene \emph{atp9} and all but seven of it's mitochondrial tRNA genes. The mechanistic details of the loss of \emph{atp9} and it potential relocation to the nucleus have not been fully explored as no other sponge species displays the same loss.

To this end, we report the publication of the mitochondrial genomes of two species, \emph{Niphates digitalis} and \emph{Niphates erecta}, both of which are most closely related to \emph{Amphimedon queenslandica} and have lost \emph{atp9}, all but four mt-tRNA genes, and an additional mt-gene. In addition, the mt-genomes of \emph{N. digitalis} and \emph{N. erecta} show drastic differences, despite their highly similar gene content, structure, and organization, as well as their close phylogenetic relationship. These two genomes, when compared with \emph{Amphimedon queenslandica}, give the history of the loss and movement of \emph{apt9} in better detail and provides an interesting case study on the rapid invasion of stem-loops in sponges.

\end{document}