\documentclass[../main.tex]{subfiles}
\begin{document}

\newpage
\section{Introduction}

The mitochondrial genomes of non-bilaterian animals, such as sponges, ctenophores, and cnidarians hold the majority of the genetic diversity seen in Metazoan mitochondrial genomes. Of these, phylum Porifera (sponges) offers a unique perspective on ancient Metazoans and the subsequent evolution of key animal traits such as symmetry and tissue development. Their ancient evolutionary history, combined with their wide geographically range and availability make them useful for studying the genetic diversity in non-bilaterians. The most well-studied and largest group in phylum Porifera is Class Demospongiae - referred to simply as demosponges from here on out - which represents over 80\% of all sponge species on the Earth.

Previous studies on sponge mt-genomes have shown remarkable variation in terms of their gene structure, gene content, size, and organization. Much of this variation is lineage-specific. One of the more novel traits discovered is the repeated loss of mt-tRNA genes in many sponge species. 

[Insert paragraph with information about tRNAs and tRNA genes here, for background]

However, the loss of mt-tRNA genes in sponges is sporadic and not lineage specific. For all the sponge species with sequenced mitochondrial genomes, no pattern of loss has been identified, and the impact the loss has on cellular processes and genome evolution has not been widely studied. Previous research has shown mitochondrial tRNA synthetases can be lost in parallel with mitochondrial tRNAs genes. However, this is not always the case. It does appear that certain mt-tRNA genes and mitochondrial tRNA synthetases are retained. The factors that promote this loss of tRNA genes and the mechanisms by which it occurs are also unknown. 

This study represents an on-going effort to assess the systemic effects of mt-tRNA gene loss in sponges. To this end, we report the publication of the mitochondrial genomes of two closely related species, \emph{Niphates digitalis} and \emph{Niphates erecta}, both of which have lost all but four mt-tRNA genes. Despite their close phylogenetic relationship, the two mt-genomes show striking differences in genome architecture, including large insertions, novel stem-loop elements, and gene rearrangement. 

\end{document}