\documentclass[../main.tex]{subfiles}
\begin{document}

\section{Results}

\subsection{Genome description}
The two new genomes generated for this study were found to be strikingly similar. Both are circular-mapping molecules, and contain 11 protein-coding gene, two rRNA genes, and four tRNA genes. Both species are missing \emph{atp8} and \emph{atp9}, as is common in other demosponges. The gene order for both species is the same, as seen in Figure 1, but show a difference in length. The \emph{N. erecta} and \emph{N. digitalis} genomes are 25525 bps and 18285 bps, respectively. The majority of this length difference can be attributed to the increased number of intergenic regions seen in \emph{N. erecta}. Intergenic regions account for 24.2\% of \emph{N. erecta}'s genome, as opposed to 13.77\% in \emph{N. digitalis}. 

The two genomes also show an identical nucleotide composition. The \emph{N. erecta} genome has a GC and AT content of 36.40\% and 63.65\%, while \emph{N. digitalis} has a GC and AT content of 36.40\% and 63.60\%. 

Overall, \emph{N. digitalis} displays a much more compacted mt-genome structure as compared to \emph{N. erecta}, with fewer intergenic regions, but the same genome composition and organization.

\subsection{Insertions}
The primary difference between the two genomes lies in the novel insertions identified in the \emph{N. erecta} mt-genome. When compared to \emph{N. digitalis, N. erecta} has a multitude of insertions present. Eighty insertions have been identified. These insertions were run through NCBI BLAST and found to not be contamination, and do not have any close sequence relation. The size of insertions ranged in size from a few base pairs to large sections up to 650 bps long, and have a similar nucleotide composition as the genome as whole. Additionally, the majority of insertions fall into intergenic regions. The location of all insertions can be seen in Figure 1. Though insertions were not exclusively found in intergenic regions, as genes also showed novel insertions - \emph{cox2, nad1, nad5, rnl}, and \emph{rns}. 

This led to the question of if these insertions were present in downstream products. cDNA analysis of genes with insertions showed that...

This indicates...

\subsection{Inverted Repeats}

Further analysis of \emph{N. erecta}'s insertions found that many contain inverted repeats. The locations of these can be seen in Figure 1. Alignments of the inverted repeats showed that majority of repeats have the same general sequence, see Figure 2, and when folded form hairpin-like structures. The function of these insertions in the genome is unknown, but other inverted repeats have been seen in other sponge species. 

\subsection{tRNA content and sythetases}
One of the novel traits seen in many sponge mitochondrial genomes is a lack of mitochondrial tRNAs. \emph{Niphates digitalis} and \emph{Niphates erecta} are no exceptions. Both genomes had the same four mitochondrial tRNAs - Y, I, M, and W. These tRNAs fall into the same place in the genome, with Y, I, and M between \emph{nad4l} and \emph{cox2}, and W being between \emph{cob} and \emph{rns}.

Alignment of these tRNA genes with mt-tRNA genes from other demosponge species shows these tRNAs to be closely related to the same tRNA gene in other species, with no abnormal structures. 

Previous research has shown that 

Viraj is looking into synthetases to see how many are still present.

\subsection{Accelerated sequence evolution}
When protein-coding sequences were aligned with other demosponge species, both species showed an accelerated rate of evolution. A long branch connected \emph{N. digitalis} and \emph{N. erecta} to others of their group, and the analysis gave a short evolutionary distance between the two species. Interestingly, \emph{N. digitalis} has a slightly higher rate of evolution as compared to \emph{N. erecta}, even though \emph{N. erecta} has additional insertions.

\end{document}