\documentclass[../main.tex]{subfiles}
\begin{document}

\section{Results}

\subsection{Genome description}
The two new genomes generated for this study were found to be strikingly similar. Both are circular-mapping molecules, and contain 11 protein-coding genes, two rRNA genes, and four tRNA genes. Both species are missing \emph{atp8} and \emph{atp9}. In previously published phylogenies, both species of \emph{Niphates} are most closely related to \emph{Amphimedon queenslandica}. \emph{Amphimedon queenslandica} has a mitochondrial genome that codes for 13 proteins, and lacks \emph{atp9}, but retains \emph{atp8} unlike either \emph{Niphates} species. Functional transcripts of \emph{atp8} and \emph{atp9} were found the transcriptome of \emph{Niphates digitalis}, suggesting a move to the nuclear genome reminiscent of \emph{apt9}'s move to the nuclear genome in \emph{Amphimedon queenslandica}. An analysis of the \emph{atp9} nucleotide sequences from \emph{Niphates digitalis} and \emph{Amphimedon queenslandica} as compared to \emph{atp9} sequences in other species showed...

The gene order for both species of \emph{Niphates} species is the same, as seen in Figure 1, but is incredibly different when compared to all other demosponges. In other demosponges, the ribosomal subunit genes \emph{rns} and \emph{rnl} are usually found together, generally separated by tRNA genes for valine and glycine and some others. In both \emph{Niphates} species, \emph{rnl} and \emph{rns} are found far apart, with multiple genes and genomic features in between. Another trend seen in demosponges is the gene order of \emph{cox2 - tRNAs - atp8 - atp6 - tRNAs - cox3 - tRNAs - cob - atp9}. This is also not seen in either \emph{Niphates} species. As mentioned above, \emph{atp9} and \emph{atp8} are missing entirely, though \emph{atp6} has been retained. The genes \emph{cox1} and \emph{nad1} are positioned between \emph{cox3} and \emph{cob}. Short of \emph{nad4l} and \emph{nad1}, all NADH subunits are positioned together. In demosponge groups G3 and G4, NADH genes are separated into two groups, generally separated in gene order. \emph{Niphates} species do not follow this trend. 

One of the differences between the two \emph{Niphates} genomes is their length. The \emph{N. erecta} and \emph{N. digitalis} genomes are 25525 bps and 18285 bps, respectively. The majority of this length difference can be attributed to the increased number of intergenic regions seen in \emph{N. erecta}. Intergenic regions account for 24.2\% of \emph{N. erecta}'s genome, as opposed to 13.77\% in \emph{N. digitalis}. The two genomes also show an identical nucleotide composition. The \emph{N. erecta} genome has a GC and AT content of 36.40\% and 63.65\%, while \emph{N. digitalis} has a GC and AT content of 36.40\% and 63.60\%. 


\subsection{Insertions}
The primary difference between the two genomes lies in the novel insertions identified in the \emph{N. erecta} mt-genome. When compared to \emph{N. digitalis, N. erecta} has a multitude of insertions present. Eighty insertions have been identified. Insertion sequences were analyzed using NCBI BLAST and found to have no close sequence relation, regardless of parameters. This discounts these insertions as possible contamination. The size of insertions ranged in size from a few base pairs to large sections up to 650 bps long, and have a similar nucleotide composition to the genome as whole. Additionally, the majority of insertions fall into intergenic regions, as shown in Figure 1. Though insertions were not exclusively found in intergenic regions, as genes also showed novel insertions - \emph{cox2, atp6, nad1, nad5, rnl}, and \emph{rns}, of which \emph{cox2, nad1,} and \emph{nad5} are protein-coding.

As a few of these insertions fall into protein-coding genes, it was important to understand the impact these insertions are having on down-stream products. To begin, the insertions were categorized on if they are in-frame or out-of-frame. There are four insertions found in \emph{cox2} and all are in-frame insertions, and the overall nucleotide addition to \emph{cox2} is in-frame. The gene \emph{atp6} has 5 insertions, one of which is out-of-frame. The overall nucleotide addition to \emph{atp6} is also out-of-frame. Three of the genes involved in NADH production, \emph{nad1, nad2,} and \emph{nad5} also have insertions - \emph{nad1} has two insertions, both of which are in-frame, \emph{nad2} has one insertion, which is in-frame, and \emph{nad5} has four insertions. Three of the four insertions are out-of-frame, and the overall nucleotide addition is out-of-frame for \emph{nad5}. 

As two of these genes - \emph{atp6} and \emph{nad5} - have out-of-frame insertions, the next step was to determine if these insertions are present in downstream products. **INSERT TRANSCRIPTOME ANALYSIS HERE WHEN COMPLETED**

**Also maybe talk about the deletions in N. erecta was compared to N. digitalis.

\subsection{Inverted Repeats}

Further analysis of \emph{N. erecta}'s insertions and intergenic regions found that many contain inverted repeat sequences. The locations of these can be seen in Figure 1.  Inverted repeats have previously been identified in other sponge species, and their function is unknown. These inverted repeats were isolated from the genome, and like the insertions, analyzed with NCBI BLAST. They returned no close sequence relations under any parameters. Alignments of the \emph{Niphates} inverted repeats showed that majority of repeats have the same general sequence, see Figure 2, and when folded form hairpin-like structures.

As previously seen in demosponges, the inverted repeats can be categorized into three different motifs, the structure of which can be seen in Figure 2. The majority of inverted repeats contain motif 1. Motif 1 contains a highly conserved 11-nucleotide stem with a less conserved 4 nucleotide loop. Before and after the stem-loop is not conserved. Motif 2 shows a similar pattern, though it is less common. It contains a well-conserved 12 nucleotide stem with a 3 nucleotide loop at the top. In addition, bases before the stem-loop are typically conserved. Motif 3 is much smaller, with a 5 nucleotide stem and a 6 nucleotide loop. All three of these motifs 

\subsection{tRNA content and sythetases}
One of the novel traits seen in many sponge mitochondrial genomes is a lack of mitochondrial tRNAs. \emph{Niphates digitalis} and \emph{Niphates erecta} are no exceptions. Both genomes had the same four mitochondrial tRNAs - Y, I, M, and W. These tRNAs fall into the same place in the genome, with Y, I, and M between \emph{nad4l} and \emph{cox2}, and W being between \emph{cob} and \emph{rns}.

Alignment of these tRNA genes with mt-tRNA genes from other demosponge species shows these tRNAs to be closely related to the same tRNA gene in other species, with no abnormal structures. 

Previous research has shown that 

Viraj is looking into synthetases to see how many are still present.

\subsection{Accelerated sequence evolution}
When protein-coding sequences were aligned with other demosponge species, both species showed an accelerated rate of evolution. A long branch connected \emph{N. digitalis} and \emph{N. erecta} to others of their group, and the analysis gave a short evolutionary distance between the two species. Interestingly, \emph{N. digitalis} has a slightly higher rate of evolution as compared to \emph{N. erecta}, even though \emph{N. erecta} has additional insertions.

\end{document}